imagepicker/build.gradle

```diff
@@ -46,15 +45,10 @@ artifacts {
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
 
-    final glideVersion = '4.5.0'
-    implementation "com.github.bumptech.glide:glide:$glideVersion"
+    implementation "com.github.bumptech.glide:glide:4.5.0"
 
-    final supportLibraryVersion = '27.1.1'
-    implementation "com.android.support:recyclerview-v7:$supportLibraryVersion"
-    implementation("com.android.support:appcompat-v7:$supportLibraryVersion") {
-        exclude module: 'animated-vector-drawable'
-        exclude module: 'support-media-compat'
-    }
+    implementation "androidx.recyclerview:recyclerview:1.0.0"
+    implementation "androidx.appcompat:appcompat:1.0.0"
```

Comment:
The support-media-compat module is not a dependency anymore is but animated-vector-drawable is. I can remove it but there so many other dependencies imported by androidx.appcompat:appcompat that it would be a small fish in the ocean. Figuring out all the dependencies that needs to be removed was outside of my scope.

Reply:
I guess you're right